.travis.yml

```diff
@@ -17,9 +17,9 @@ jobs:
     # "provisional feature" rules there are significant differences between patch
     # versions for asyncio and typing.
     - python: 3.5.2
-    - python: '3.5'
-    - python: '3.6'
-    - python: '3.7'
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
```

Comment:
Those quotes are actually necessary. Otherwise these get interpreted as floats.

Reply:
The quotes are meaningful, but they're apparently not necessary - Travis manages to get the right Python versions even with this patch. However, I think if we're going to make a change here it's better to go the other direction: Add quotes to all the values to make them all unambiguously strings, since they are all semantically strings instead of floats.